player/lua/stats.lua

```diff
@@ -1749,14 +1750,20 @@ mp.add_key_binding(nil, "display-stats", function() process_key_binding(true) en
 mp.add_key_binding(nil, "display-stats-toggle", function() process_key_binding(false) end,
     {repeatable=false})
 
--- Single invocation bindings without key, can be used in input.conf to create
--- bindings for a specific page: "e script-binding stats/display-page-2"
 for k, _ in pairs(pages) do
-    mp.add_key_binding(nil, "display-page-" .. k,
-        function()
-            curr_page = k
-            process_key_binding(true)
-        end, {repeatable=true})
+    -- Single invocation key bindings for specific pages, e.g.:
+    -- "e script-binding stats/display-page-2"
+    mp.add_key_binding(nil, "display-page-" .. k, function()
+        curr_page = k
+        process_key_binding(true)
+    end, {repeatable=true})
+
+    -- Key bindings to toggle a specific page, e.g.:
+    -- "e script-binding stats/display-page-4-toggle".
```

Comment:
```suggestion
    -- "h script-binding stats/display-page-4-toggle".
```

Reply:
Yeah I think it wouldn't be bad to even make this, or ?, a default binding later. When page 4 was added one of the reasons we didn't bind it by default is that we would have to explain to the user how to close it but actually using it pressing h again to close seems intuitive to me at least.

Reply:
Could we bind esc to close, I think it is safe key when user don't know what to do.